oap-server/server-starter/src/main/resources/component-libraries.yml

```diff
@@ -425,6 +425,15 @@ tls:
 Micronaut:
   id: 131
   languages: Java
+AWSDynamoDB:
+  id: 132
+  languages: Java,C#,Node.js,Python
+AWSSNS:
+  id: 133
+  languages: Java,C#,Node.js,Python
+AWSSQS:
+  id: 134
+  languages: Java,C#,Node.js,Python
```

Comment:
132-134 have been registered. Please register to new IDs.

Reply:
Fixed.